src/repl-window/repl-history.ts

```diff
@@ -142,9 +142,26 @@ function showNextReplHistoryEntry(): void {
   }
 }
 
+function searchReplHistory(): void {
+  const editor = util.getActiveTextEditor();
+  const history = getHistory(getSessionType());
+  vscode.window.showInputBox({
```

Comment:
It may be this dangling promise that causes the linter to complain. I think the solution is to return the promise, because otherwise systems like Joyride and `runCommands` can't wait for the command to finish, which limits its usefulness.

```suggestion
  return vscode.window.showInputBox({
```